src/sbt-test/universal/multiproject-classifiers/build.sbt

```diff
@@ -25,18 +27,20 @@ lazy val sub = project
     ivyConfigurations += Assets,
     Assets / artifact := artifact.value.withClassifier(classifier = Some("assets")),
     packagedArtifacts += {
+      implicit val converter: FileConverter = fileConverter.value
       val file = target.value / "assets.jar"
       val assetsDir = baseDirectory.value / "src" / "main" / "assets"
       val sources = assetsDir.**(AllPassFilter).filter(_.isFile) pair (file => IO.relativize(assetsDir, file))
       IO.zip(sources, file)
-      (Assets / artifact).value -> file
+      (Assets / artifact).value -> PluginCompat.toFileRef(file)
     },
     Assets / exportedProducts := {
-      Seq(
-        Attributed
-          .blank(baseDirectory.value / "src" / "main" / "assets")
-          .put(artifact.key, (Assets / artifact).value)
-          .put(AttributeKey[ModuleID]("module-id"), projectID.value)
+      implicit val converter: FileConverter = fileConverter.value
+      val assetsDir = baseDirectory.value / "src" / "main" / "assets"
+      assetsDir.**(AllPassFilter).filter(_.isFile).classpath.map(
+        _
+          .put(PluginCompat.artifactStr, PluginCompat.artifactToStr((Assets / artifact).value))
+          .put(PluginCompat.moduleIDStr, PluginCompat.moduleIDToStr(projectID.value))
```

Comment:
If I recall correctly, this was migrated from using a directory ( `src/main/assets` ) to all the files in the directory, because the new `HashedVirtualFileRef` cannot be used with directories...